ext/spl/spl_directory.c

```diff
@@ -2913,7 +2913,7 @@ SPL_METHOD(SplFileObject, fread)
 	}
 
 	if (length <= 0) {
-		php_error_docref(NULL, E_WARNING, "Length parameter must be greater than 0");
+		php_error_docref(NULL, E_WARNING, "length parameter must be greater than 0");
```

Comment:
Wrong way around?

Reply:
My best guess why I did is that `length` is a parameter name - which should be lowercase. I'd like to improve this wording in the future though.